### PR TITLE
[API] Add simple item to cart

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | Related to #11250, Replaces #11528
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Service/ApiSecurityService.php [hunk 1]
```diff
@@ -26,24 +27,38 @@ final class ApiSecurityService implements SecurityServiceInterface
...
+        $response = $this->client->getResponse();
+        $content = json_decode($response->getContent(), true);
+
+        Assert::keyExists(
+            $content,
+            'token',
+            SprintfResponseEscaper::provideMessageWithEscapedResponseContent('Token not found.', $response)
+        );
+
+        $token = $content['token'];
```
Comment (Zales0123) [inline src/Sylius/Behat/Service/ApiSecurityService.php:61]: Is the variable needed?

Comment (Zales0123) [inline src/Sylius/Behat/Service/ApiSecurityService.php:61]: Is the variable needed?

File: src/Sylius/Behat/Service/SprintfResponseEscaper.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    public static function provideMessageWithEscapedResponseContent(string $message, Response $response): string
+    {
+        return sprintf(
+            '%s Received response: %s',
+            $message,
+            str_replace(
+                '%',
+                '%%',
+                $response->getContent()
+            )
```
Comment (Zales0123) [inline src/Sylius/Behat/Service/SprintfResponseEscaper.php:29]: This `str_replace` in one line would be much more readable for me

File: src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/AddItemToCartHandler.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler\Cart;
```
Comment (Zales0123) [inline src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/AddItemToCartHandler.php:3]: license block 💃

File: src/Sylius/Bundle/ApiBundle/DataTransformer/AddItemToCartInputDataTransformer.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
```
Comment (Zales0123) [inline src/Sylius/Bundle/ApiBundle/DataTransformer/AddItemToCartInputDataTransformer.php:3]: Here as well 🚀

File: src/Sylius/Bundle/ApiBundle/Resources/config/services.xml [hunk 5]
```diff
@@ -66,5 +66,10 @@
         <service id="sylius.api.property_info.extractor.empty_list_extractor" class="Sylius\Bundle\ApiBundle\PropertyInfo\Extractor\EmptyPropertyListExtractor">
             <tag name="property_info.list_extractor" priority="-2000" />
         </service>
+
+        <service class="Sylius\Bundle\ApiBundle\DataTransformer\AddItemToCartInputDataTransformer" id="sylius_bundle_api.data_transformer.add_item_to_cart_input_data_transformer">
+            <tag name="api_platform.data_transformer"/>
+        </service>
+
```
Comment (Zales0123) [inline src/Sylius/Bundle/ApiBundle/Resources/config/services.xml:73]: Extra blank line

**Comment by GSadee** [comment]
Thanks, Łukasz! :1st_place_medal:

**Pull request merged**